experimental/torch_xla2/torch_xla2/__init__.py

```diff
@@ -1,14 +1,24 @@
 import jax
+from jax._src import config
+import os
 import torch
 from torch._functorch import make_functional
 from torch.utils import _pytree as pytree
 from torch_xla2 import export, tensor, tf_integration
 
 jax.config.update('jax_enable_x64', True)
 
+config.update(
```

Comment:
qq. just jax.config.update won't work? must use config from _src?

Reply:
btw if you want to land now you can add a `if jax.version >= 0.4.29'` so it will work when jax releases on Monday.

Reply:
> btw if you want to land now you can add a `if jax.version >= 0.4.29'` so it will work when jax releases on Monday.

So does it mean that if I don't add `if jax.version >= 0.4.29'`, it won't work even if jax releases on Monday?